multimedia/mlt/Makefile

```diff
@@ -11,105 +11,85 @@ COMMENT?=	Multimedia framework for TV broadcasting
 LICENSE?=	GPLv2 GPLv3 LGPL21
 LICENSE_COMB?=	multi
 
-USES+=		alias compiler:c++11-lang eigen:3 gmake gnome localbase \
-		pkgconfig
-GNU_CONFIGURE=	yes
-CONFIGURE_ARGS+=--enable-gpl --enable-gpl3 --target-os=FreeBSD
+USES+=		alias cmake compiler:c++11-lang eigen:3 gnome pkgconfig
 
-.ifndef(WITH_DEBUG)
-CONFIGURE_ARGS+=--disable-debug
-.else
-CONFIGURE_ARGS+=--enable-debug
-.endif
-
-.ifndef(SLAVEPORT) # Master port.
+# It's still mlt, but installs support files to mlt-7
+DATADIR=	${PREFIX}/share/${PORTNAME}-7
 
 # They both install bin/melt.
-CONFLICTS_INSTALL=	freeze-[0-9]*
+CONFLICTS_INSTALL=	freeze-[0-9]* \
+			mlt-6*
 
 LIB_DEPENDS=	libfftw3.so:math/fftw3 \
+		libfreetype.so:print/freetype2 \
 		libogg.so:audio/libogg \
 		libvorbis.so:audio/libvorbis
 
-BROKEN_sparc64=		does not compile on sparc64 (invokes i386 asm)
-
 USE_GL=		gl
 USE_GNOME=	libxml2
 USE_SDL=	image2
 USE_XORG=	x11
 USES+=		gl iconv pathfix sdl xorg
 PATHFIX_MAKEFILEIN=	Makefile
-CONFIGURE_ARGS+=--disable-qt --swig-languages=none
 USE_LDCONFIG=	yes
 
+# Explicitly disabled modules, and explicitly always-enabled
+CMAKE_OFF=	MOD_NDI MOD_OPENCV MOD_RUBBERBAND MOD_SDL1
+CMAKE_ON=	\
+		MOD_DECKLINK MOD_KDENLIVE MOD_NORMALIZE \
+		MOD_OLDFILM MOD_PLUS MOD_PLUSGPL \
+		MOD_RTAUDIO MOD_SDL2 MOD_VORBIS MOD_XINE MOD_XML
+
 PLIST_SUB=	PORTVERSION="${PORTVERSION}"
 
-PORTDOCS=	AUTHORS NEWS README docs
+PORTDOCS=	AUTHORS NEWS README.md docs
```

Comment:
these are part of the conflict between mlt6 and ml7 -- make them be installed in a versioned dir